src/settings.rs

```diff
@@ -303,6 +313,30 @@ impl Settings {
             val.to_owned()
         });
 
+        // Setup the web-ui source based on whether we're using bundled or not
+        if let Some(bundle) = &BUNDLED_UI {
+            tracing::info!("This binary was built with a bundled web UI.");
```

Comment:
I would probably make this just a debug print, as it will print every time the application is run and isn't that important for the user to see. If there's no UI bundled they'll see that if they try to view it.